serverless.yml

```diff
@@ -13,9 +13,29 @@ provider:
   region: ap-southeast-2
 
 functions:
-  index:
-    handler: src/handler.index
+  logCloudFormationStack:
+    handler: src/logCloudFormationStack.index
     events:
-      - http:
-          method: get
-          path: index
+      - cloudwatchEvent:
+          event:
+            source:
+              - "aws.cloudformation"
+            detail:
+              eventSource:
+                - "cloudformation.amazonaws.com"
+              eventName:
+                - "CreateStack"
```

Comment:
Are there events like "create_complete" or "update_complete" that we could use. The problem being that if the stack fails to provision, we would also need to remove it from the Dynamo table.

Reply:
Nice one! I can approve this now, but we'll need a task to set up the state machine for polling the stack status